vpr/src/power/power.cpp

```diff
@@ -981,7 +981,7 @@ static void power_usage_routing(t_power_usage* power_usage,
                 /* Determine types of switches that this wire drives */
                 connectionbox_fanout = 0;
                 switchbox_fanout = 0;
-                for (t_edge_size iedge = 0; iedge < node.num_edges(); iedge++) {
+                for (t_edge_size iedge = 0; iedge < rr_graph.num_edges(rr_node); iedge++) {
```

Comment:
I cannot find where the ``rr_node`` is defined in this function. Would you mind pointing me to it?

Reply:
It is being used in the last loop of function `power_usage_routing`, and defined in line no. 861 as `RRNodeId rr_node = RRNodeId(rr_node_idx);`

Reply:
My bad. No worries. It is good.